pkg/clusteragent/clusterchecks/dispatcher_nodes.go

```diff
@@ -122,3 +122,26 @@ func (d *dispatcher) expireNodes() {
 		log.Warn("No nodes reporting, cluster checks will not run")
 	}
 }
+
+// updateRunnersStats collects stats from the registred
+// Cluster Level Check runners and updates the stats cache
+func (d *dispatcher) updateRunnersStats() {
+	if d.clcRunnersClient == nil {
+		log.Debug("Couldn't instantiate a CLC Runner client")
```

Comment:
This will be run every `runnerStatsTicker`, I think the message could be something like
`Cluster Level Check runner client was not correctly initialised` or something like that

Reply:
makes sense 👍